README.md

```diff
@@ -32,6 +32,8 @@ To perform hands-on on **go-examples** it is prerequisite to have Go installed o
 * [Struct](https://github.com/sagar-jadhav/go-examples/blob/master/src/struct.go)
 * [Switch Statement](https://github.com/sagar-jadhav/go-examples/blob/master/src/switch.go)
 * [Unit Test](https://github.com/sagar-jadhav/go-examples/tree/master/src/test/power_test.go)
+* [Variadic Parameter](https://github.com/sagar-jadhav/go-examples/blob/master/src/variadic.go)
```

Comment:
Update link as `./variadic.html`

Reply:
I do not understand your comment.
BTW, I've not made any changes regarding the above-mentioned line. @sagar-jadhav

Reply:
I agree suggested change is not related to your **change set**. 

I suggested this change to fix the broken link in README.

Reply:
Do you want me to create separate issue for this?

Reply:
No, it was a small change so I've fixed it.